### ISSUE TITLE
Missing lfs-stored files in source tarballs leads to failing tests

### ISSUE THREAD

**Issue by mzf-guest** [f3d-app/f3d#136]
**Describe the bug**
Tests are failing when using F3D built from tarball source, due to missing lfs-stored file in the generated archive. The documentation generation would likely to fail too (I haven't tried it).

**To Reproduce**
 1. Download the [1.2.0 tarball](https://github.com/f3d-app/f3d/archive/refs/tags/v1.2.0.tar.gz)
 2. untar the archive in an empty folder
 2. build the software with cmake, then `make`
 3. run test with `make test`
 4. most of the tests are failing with messages like `f3d-1.2.0/data/testing/mb.vtm does not exist`

**Expected behavior**
Test are passing and documentation is generated when using official tarball from scratch.

**System Information:**
 - OS: Debian unstable
 - GPU and GPU driver: not relevant here

**F3D Information**
Paste the content of `f3d --version`:
```
f3d 1.2.0

F3D - A fast and minimalist 3D viewer
Version: 1.2.0
Build date: 2021-12-11 23:20:48
System: Linux-5.14.0-3-amd64 x86_64
Compiler: GNU 11.2.0
RayTracing module: OFF
Exodus module: ON
OpenCASCADE module: OFF
Assimp module: OFF
VTK version: 9.0.3 (build 3)

Copyright (C) 2019-2021 Kitware SAS.
Copyright (C) 2021-2022 Michael Migliore, Mathieu Westphal.
License BSD-3-Clause.
Written by Michael Migliore, Mathieu Westphal and Joachim Pouderoux.
```

**Additional context**
Please include the lfs-stored files in the source tarballs in order to build the software without depending on any third-party website like "lfs on GitHub". In addition, git lfs is hard to configure (sometimes impossible) with other git forges so I would recommend to not force the usage.

Looks like GitHub have an option for that: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-git-lfs-objects-in-archives-of-your-repository

Thanks!

**Comment by mwestphal** [comment]
what if we had the data in a separate archive ?

**Comment by mzf-guest** [comment]
Extra work for the package maintainers, but OK for me.

**Comment by Meakk** [comment]
@mzf-guest we have enabled LFS in tarballs, it should be good now.

**Comment by mzf-guest** [comment]
Works now, thanks for the quick update!

**Comment by mwestphal** [comment]
@mzf-guest was had to turn the option off because of crawlers downloading our release repeatedly and eating all our LFS bandwidth. We can provide a secondary package containing the lfs data separately, would that work for you ?

**Comment by mzf-guest** [comment]
> [@mzf-guest](https://github.com/mzf-guest) was had to turn the option off because of crawlers downloading our release repeatedly and eating all our LFS bandwidth. We can provide a secondary package containing the lfs data separately, would that work for you ?

Yes, providing a data-package sounds like an acceptable trade-off. Thanks for taking downstream packaging into account!

**Comment by mwestphal** [comment]
@mzf-guest please check the last nightly to make sure this works for you: https://github.com/f3d-app/f3d/releases/tag/nightly

It contains a `f3d-lfs-data.zip` archive that you should be able to place in the source.
